fabfile.py

```diff
@@ -53,6 +53,26 @@ def gitrepos():
             run("git checkout -t origin/maintenance/1.10.x")
             run("git submodule update")
 
+def prepare_scipy():
+    run("mkdir -p repos")
+    with cd("repos"):
+        run("git clone https://github.com/scipy/scipy")
+        with cd("scipy"):
+            run("git submodule init")
+            run("git submodule update")
+
+    sudo("sudo apt-get install libatlas-base-dev")
```

Comment:
A minor quibble: this clones from the main scipy repo which is not necessarily convenient. Maybe there's a way of giving it a `fork="scipy"` parameter so that the git clone path is `"github.com/" + fork + "/scipy" and expose this as a command-line arg?